src/app/payment/views/IntegratedCheckoutView/CheckoutView.tsx

```diff
@@ -21,7 +23,7 @@ export const PAYMENT_ELEMENT_OPTIONS: StripePaymentElementOptions = {
   },
   layout: {
     type: 'accordion',
-    defaultCollapsed: false,
+    defaultCollapsed: false, // Set to true when the crypto payment is enabled
```

Comment:
nit: why when crypto payments is enabled this need to be collapsed by default? in order to let user open which prefers?

Reply:
Yes. It is better to contract so that the user can choose between Stripe Elements (card or PayPal) and cryptocurrencies (bit2me provider).